libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -299,6 +299,8 @@ static bool _mqttSubscription_setUnsubscribe( const IotLink_t * const pSubscript
      * must never be NULL. */
     IotMqtt_Assert( pSubscriptionLink != NULL );
 
+    /* coverity[misra_c_2012_rule_20_7_violation] */
+    /* coverity[caretline] */
```

Comment:
Can we have a comment above this about why these suppressions are necessary? It can be copy pasted to the other eight as well.

Reply:
ok, let me add them